decidim-core/app/controllers/concerns/decidim/locale_switcher.rb

```diff
@@ -53,7 +53,7 @@ def available_locales
       #
       # Returns a String with the default locale.
       def default_locale
-        @default_locale ||= current_organization.default_locale
+        @default_locale ||= (current_organization || Decidim).public_send(:default_locale)
```

Comment:
This is ugly, but when calling `redirect_to decidim_system.root_path unless organization`, this gets called from `default_url_options`... So I had to handle to "no organization" case. Suggestions to do this better welcome.